### PR TITLE
export default toast to reuse them with custom props.

### PR DESCRIPTION
**Pull request by MrLibya**
Hello,
I want to use the default toast layout but with small changes so instead of re-create the layout I want to use the old one and just override the props.

### CODE REVIEW

**Comment by CalebLovell** [comment]
I second this PR! Can this be merged? I realized today there is no way to just extend the existing components, you have to remake it yourself. I would really just like to increase the font size of the text and leave the rest as is, but right now there is no way to do that. This PR would basically fix that...

File: README.md [hunk 1]
```diff
@@ -114,11 +114,11 @@ const props = {
 
 If you want to add custom types - or overwrite the existing ones - you can add a `config` prop when rendering the `Toast` in your app `root`. 
 
-You can either use the default `BaseToast` style and adjust its layout, or create Toast layouts from scratch.
+You can either use the default `BaseToast` style and adjust its layout, or create Toast layouts from scratch or reuse the default layout
```
Comment (calintamas) [inline README.md:117]: Can you adjust the wording please?

```suggestion
You can either use any of the default `BaseToast`, `SuccessToast`, `ErrorToast` or `InfoToast` components and adjust their layout, or create Toast layouts from scratch.
```

File: README.md [hunk 2]
```diff
@@ -139,6 +139,20 @@ const toastConfig = {
...
+        <ErrorToast {...rest} onTrailingIconPress={hide} 
+            text1Style={{
+                fontSize: 17
+            }}
+            text2Style={{
+                fontSize: 15
+            }}
+        />
+    ),
+
```
Comment (calintamas) [inline README.md:155]: Here as well

```suggestion
   /*
    Reuse the default ErrorToast toast component
  */
  error: (props) => (
    <ErrorToast 
      {...props}
      text1Style={{
        fontSize: 17
      }}
      text2Style={{
        fontSize: 15
      }}
    />
  ),
```

File: index.js [hunk 3]
```diff
@@ -1,4 +1,7 @@
 import Toast from './src';
 
 export { default as BaseToast } from './src/components/base';
+export { default as SuccessToast } from './src/components/success';
+export { default as ErrorToast } from './src/components/error';
+export { default as InfoToast } from './src/components/info';
```
Comment (calintamas) [inline index.js:6]: If we are to do this, the type definition file would need to be updated as well
https://github.com/calintamas/react-native-toast-message/blob/master/index.d.ts

**Review by calintamas — CHANGES_REQUESTED** [review]
@MrLibya Thanks for your contribution! I've added a few small comments, nothing major

**Comment by calintamas** [comment]
> I second this PR! Can this be merged? I realized today there is no way to just extend the existing components, you have to remake it yourself. I would really just like to increase the font size of the text and leave the rest as is, but right now there is no way to do that. This PR would basically fix that...

@CalebLovell Yes, the type definition file needs to be updated before I can merge, but other than that, all good

**Comment by MrLibya** [comment]
@calintamas 
I've updated the type as well, tbh I dunno if it could be done in another way so plz check it :)

**Review by calintamas — APPROVED** [review]
Looks good, thanks!

**Comment by calintamas** [comment]
Tests workflow failing due to coveralls issue (all tests pass), I'll merge

**Comment by CalebLovell** [comment]
Thanks for doing this, added it today to our app and it works great!

**Pull request merged**